framework/doc/content/source/transfers/MultiAppInterpolationTransfer.md

```diff
@@ -2,7 +2,16 @@
 
 The MultiAppInterpolationTransfer transfers the source variable to the nearest node on the
 target mesh using mesh interpolation, including the ability to utilize the displaced
-configuration for either or both the source and target.
+configuration for either or both the source and target.  
+
+Nodal transfers using this interpolation scheme is best suited for the interpolation of point cloud data in the source mesh onto a target mesh.  
+The [MultiAppMeshFunctionTransfer](/MultiAppMeshFunctionTransfer.md) may be a better choice for nodal transfers between two meshes because element data will be used in the transfer and the transfer is not dependent on the number of points used in the transfer.  
+Using it for mesh to mesh nodal transfers of data can lead to a parallel inconsistency that arises when the point being interpolated to on the
```

Comment:
What is it referring to?
I imagine InterpolationTransfer but would be helpful to repeat to avoid any confusion